Sources/Drain.swift

```diff
@@ -1,4 +1,8 @@
-public final class Drain: DataRepresentable, Stream {
+public final class Drain: DataRepresentable, BinaryStream {
+    
+    public typealias Received = Data
+    public typealias Sent = Data
```

Comment:
I don't understand this? `Sent` and `Received` sound like they're going to be `Int` types. What's wrong w/ just using `Data`?